client/python/ovmsclient/lib/Makefile

```diff
@@ -15,10 +15,11 @@
 #
 
 VIRTUALENV_EXE := python3 -m virtualenv -p python3
-VIRTUALENV_DIR := .venv
+VIRTUALENV_DIR := .venv-ovmsclient
 ACTIVATE="$(VIRTUALENV_DIR)/bin/activate"
 
-PACKAGE_PATH := dist/ovmsclient-2022.2-py3-none-any.whl
+PACKAGE_PATH := dist/ovmsclient-2022.3-py3-none-any.whl
+TEST_TYPE := BASIC
```

Comment:
What's the difference between basic and full? Is it explained in some doc?

Reply:
Basic runs with latest (default) versions dependencies for the environment. Full also reruns tests but with reinstalled versions of dependencies that we know are significantly different than latest (different in a way that forces us to change the code to support both). For now it's for numpy 1.23.